src/uwtools/resources/jsonschema/cdeps.jsonschema

```diff
@@ -43,7 +43,10 @@
               "items": {
                 "type": "string"
               },
-              "type": "array"
+              "type": [
+                "array",
+                "string"
+              ]
```

Comment:
Accept either an array of strings or just a single string for the `stream_vectors` value.